sbt-cross-test/src/sbt-test/new-api/platformSpecific/build.sbt

```diff
@@ -0,0 +1,28 @@
+import sbtcross.{crossProject, CrossType}
+
+val g = "com.example.platformSpecific"
+val a = "bar"
+val v = "0.1.0-SNAPSHOT"
+
+lazy val bar =
+  crossProject(NativePlatform)
+    .crossType(CrossType.Pure)
+    .settings(
+      scalaVersion := "2.11.8",
+      organization := g,
+      moduleName := a,
+      version := v
+    )
+
+lazy val barNATIVE = bar.native
+
+lazy val foo =
+  crossProject(JVMPlatform, NativePlatform)
```

Comment:
You have a directory `platformSpecific/foo/js/`, but you do not cross-compile `foo` with `JSPlatform`. Is this intended? If yes, it deserves a comment.